tests/task/task_runner/test_standard_task_runner.py

```diff
@@ -62,15 +62,12 @@ def logging_and_db(self):
         (as the test environment does not have enough context for the normal
         way to run) and ensures they reset back to normal on the way out.
         """
+        clear_db_runs()
         dictConfig(LOGGING_CONFIG)
         yield
         airflow_logger = logging.getLogger('airflow')
         airflow_logger.handlers = []
-        try:
-            clear_db_runs()
-        except Exception:
-            # It might happen that we lost connection to the server here so we need to ignore any errors here
-            pass
+        clear_db_runs()
```

Comment:
Is this related? I’m suspecting the try-except is there for a reason… but we can always add it back anyway.

Reply:
Kinda. This was why we saw a failure in `TestBaseSensor.test_fail`, which just happened to be the next test that used the db. I'd rather see an exception here during cleanup instead of in an unrelated test as it took me a while to track down which test was responsible.